recipes/rank_filter/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rank_filter" %}
+{% set version = "0.4.11" %}
+{% set checksum = "246f5c73ebd03314d9b2cff3ede74330" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ checksum }}
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - boost >=1.56
+    - python
+    - setuptools >=18.0
+    - numpy x.x
+    - numpy >=1.7
+    - cython >=0.23
+
+  run:
+    - boost >=1.56
+    - python
+    - numpy x.x
+    - numpy >=1.7
```

Comment:
You're right. This is basically irrelevant. In practice, I am doing my CI against 1.10.x so I don't know how true this still is. Though I am not using any recent features from NumPy only standard things like `shape`, `reshape`, and such.